tests/connection/AConnectionTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Bolt\tests\connection;
+
+use Bolt\Bolt;
+use Bolt\connection\IConnection;
+use Bolt\connection\Socket;
+use Bolt\connection\StreamSocket;
+use Bolt\error\ConnectionTimeoutException;
+use Bolt\error\MessageException;
+use Bolt\helpers\Auth;
+use Bolt\protocol\V4;
+use PHPUnit\Framework\TestCase;
+
+final class AConnectionTest extends TestCase
+{
+    public function provideConnections(): array
+    {
+        return [
+            [StreamSocket::class],
```

Comment:
add string keys to know which one is which in phpunit result window

Reply:
Great idea, I did not know this was possible. Fixed in b19fa4c